go/vt/vtorc/config/config.go

```diff
@@ -42,94 +42,261 @@ const (
 )
 
 var (
-	sqliteDataFile                 = "file::memory:?mode=memory&cache=shared"
-	instancePollTime               = 5 * time.Second
-	snapshotTopologyInterval       = 0 * time.Hour
-	reasonableReplicationLag       = 10 * time.Second
-	auditFileLocation              = ""
-	auditToBackend                 = false
-	auditToSyslog                  = false
-	auditPurgeDuration             = 7 * 24 * time.Hour // Equivalent of 7 days
-	recoveryPeriodBlockDuration    = 30 * time.Second
-	preventCrossCellFailover       = false
-	waitReplicasTimeout            = 30 * time.Second
-	tolerableReplicationLag        = 0 * time.Second
-	topoInformationRefreshDuration = 15 * time.Second
-	recoveryPollDuration           = 1 * time.Second
-	ersEnabled                     = true
-	convertTabletsWithErrantGTIDs  = false
```

Comment:
Why did you decide not to make these two dynamic / viper-managed? I understand why you cannot make sqliteDataFile dynamic, that makes perfect sense, however, that can also be viperized and provided via the config file.

Reply:
I had originally just changed all the configurations that could be specified in the `config` file to be viper enabled. I figured what else we make viper configurable would be a discussion in itself. But I can make these two to be viper enabled too.